src/main/java/org/sagebionetworks/bridge/validators/AssessmentValidator.java

```diff
@@ -0,0 +1,64 @@
+package org.sagebionetworks.bridge.validators;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import static org.sagebionetworks.bridge.BridgeConstants.BRIDGE_EVENT_ID_ERROR;
+import static org.sagebionetworks.bridge.BridgeConstants.BRIDGE_EVENT_ID_PATTERN;
+
+import java.util.Optional;
+
+import org.springframework.validation.Errors;
+import org.springframework.validation.Validator;
+
+import org.sagebionetworks.bridge.models.OperatingSystem;
+import org.sagebionetworks.bridge.models.assessments.Assessment;
+
+public class AssessmentValidator implements Validator {
+
+    static final String CANNOT_BE_BLANK = "cannot be missing, null, or blank";
+    private final Optional<Assessment> optional;
+    
+    public AssessmentValidator(Optional<Assessment> optional) {
+        this.optional = optional;
+    }
```

Comment:
Add some comments describing what "optional" is. Why are we validating an assessment against another assessment.

Reply:
First I refactored to take the assessmentService, as discussed. Then eventually I realized we are already checking this where it matters and throwing an EntityAlreadyExistsException, so ultimately I removed all of this from the validator.